presto-cli/src/main/java/io/prestosql/cli/Console.java

```diff
@@ -153,6 +154,7 @@ public boolean run()
             exiting.set(true);
             interruptor.interrupt();
             awaitUninterruptibly(exited, EXIT_DELAY.toMillis(), MILLISECONDS);
+            closeTerminal();
```

Comment:
Please add code comment why this is needed.

Reply:
AC @findepi